### PR TITLE
Refetch Positions 3s after addliquidity

### PR DESCRIPTION
**Pull request by nickwang14**
# Description
Converting fetch to use apollo
New Atom for currentuserpositions
Fetch positions in 3s timeout after addliquiditysuccess

https://user-images.githubusercontent.com/25964487/198162574-e23852df-680f-4777-8fec-41f897365d3e.mp4

## Sanity Check
Positions are still fetched
App doesnt crash when connecting or disconnecting from wallet
Adding liquidity should refresh the myCurrentPositions

## Notes
Adding liquidity refetches positions (Should we try to periodically fetch somewhere, In case positions on perp are made elsewhere, but also in case 3s isnt enough buffer somehow by freak chance)

### CODE REVIEW

File: src/queries/getAllPerpMarketsQuery.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { gql } from '@apollo/client'
 
-export const getAllPerpMarketsQueryApollo = gql`
+export const getAllPerpMarketsQuery = gql`
```
Comment (nickwang14) [inline src/queries/getAllPerpMarketsQuery.ts:3]: Rename as we move towards Apollo fetch as standard

File: src/components/Modals/AddLiquidity/ConfirmAddLiquidity.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { useState } from 'react'
+// useRecoilRefresher_UNSTABLE
```
Comment (mattiaslightstone) [inline src/components/Modals/AddLiquidity/ConfirmAddLiquidity.tsx]: ?

File: src/components/Modals/AddLiquidity/ConfirmAddLiquidity.tsx [hunk 3]
```diff
@@ -35,9 +40,16 @@ export const ConfirmAddLiquidity = ({
...
     const totalInvestment = useRecoilValue(totalUsdInvestmentAtom)
     const currentPrice = useRecoilValue(priceInfoAtom).lastTradePriceUsd
-
     const { instance: clearingHouseInstance } = useClearingHouseContract()
 
+    const resetTotalUsdInvestmentAtom = useResetRecoilState(
+        totalUsdInvestmentAtom,
+    )
+
+    const resetUserCurrenPositions = useRecoilRefresher_UNSTABLE(
```
Comment (mattiaslightstone) [inline src/components/Modals/AddLiquidity/ConfirmAddLiquidity.tsx:49]: this is cool

**Review by mattiaslightstone — CHANGES_REQUESTED** [review]
It seems as though 3s has not been sufficient for the subgraph to have updated.

**Pull request closed without merging**